LEGEND.md

```diff
@@ -14,17 +14,19 @@
 
 ### <a name="slides">Slide Shortcuts</a>
 
-#### Mac:
+#### MacOS:
 
 Cmd + e = empty slide
+Cmd + 1 = Waheguru Slide
 
 #### Windows:
 
 Ctrl + e = empty slide
+Ctrl + 1 = Waheguru Slide
 
 ### <a name="others">Others</a>
```

Comment:
I don't think we need to include a list of all the Windows/Mac shortcuts that are commonly used outside of STTM. We should keep this to just STTM keyboard shortcuts to prevent having a massive list.